website/src/pages/docs/integrations/integration-with-express.mdx

```diff
@@ -28,12 +28,45 @@ const app = express()
 
 const yoga = createYoga()
 
-// Bind GraphQL Yoga to `/graphql` endpoint
-app.use('/graphql', yoga)
+app.use(yoga)
```

Comment:
If you do this, Yoga will show the landing page for other endpoints.

Reply:
Yes, you think it's not a good thing ?

Reply:
I though it was more aligned with other examples, most of them just renders the landing page on any endpoint.

Reply:
No, if user wants to define other routes, it won't work.

Reply:
Ok, understood. I rolledback and remvoved the section I have created.

I think it's better to use the same pattern I saw in other examples that explicitly links yoga endpoint to express enpoint.